content/glossary/sql/and.md

```diff
@@ -0,0 +1,34 @@
+<!--
+---
+Title: "AND"
+Subjects:
+  - "sql"
```

Comment:
No more need for language names in tags and subjects lists

Reply:
👍🏼